flight/PiOS.osx/osx/Libraries/FreeRTOS/Source/portable/GCC/Posix/port.c

```diff
@@ -465,7 +465,7 @@ sigset_t xSignalToBlock;
 	/* Cleanup the mutexes */
 	pthread_mutex_destroy( &xSuspendResumeThreadMutex );
 	pthread_mutex_destroy( &xSwappingThreadMutex );
-	vPortFree( (void *)pxThreads );
+	//vPortFree( (void *)pxThreads );
```

Comment:
we don't tend to have commented code. You can just delete this line and the one below.